NameError: Invalid flag: no_cfunc_wrapper when using numba 0.48.0

Exception:
```
rbc/tests/test_numpy_rjit.py::test_trunc staring rpc.thrift server: /home/pearu/git/guilhermeleobas/rbc/rbc/remotejit.thriftFAILED... stopping rpc.thrift server


================================================================================================= FAILURES =================================================================================================
________________________________________________________________________________________________ test_trunc ________________________________________________________________________________________________

rjit = <rbc.remotejit.RemoteJIT object at 0x7f81e4b53a90>

    def test_trunc(rjit):
        @rjit('double(double)')
        def trunc(x):
            return np.trunc(x)
    
>       assert (np.allclose(trunc(0.3), np.trunc(0.3)))

rbc/tests/test_numpy_rjit.py:21: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
rbc/remotejit.py:272: in __call__
    self.remotejit.remote_compile(self.func, ftype, target_info)
rbc/remotejit.py:534: in remote_compile
    [(func, [ftype])], target_info, debug=self.debug)
rbc/irtools.py:244: in compile_to_LLVM
    flags.set('no_cfunc_wrapper')
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = Flags(no_compile=True, no_cpython_wrapper=True, auto_parallel=<numba.targets.cpu_options.ParallelOptions object at 0x7...s.FastMathOptions object at 0x7f81f69e6f10>, inline=<numba.targets.cpu_options.InlineOptions object at 0x7f81f69e6f50>)
name = 'no_cfunc_wrapper', value = True

    def set(self, name, value=True):
        if name not in self.OPTIONS:
>           raise NameError("Invalid flag: %s" % name)
E           NameError: Invalid flag: no_cfunc_wrapper

../../../miniconda3/envs/rbc-dev/lib/python3.7/site-packages/numba/utils.py:311: NameError
========================================================================================= short test summary info ==========================================================================================
FAILED rbc/tests/test_numpy_rjit.py::test_trunc - NameError: Invalid flag: no_cfunc_wrapper
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================================= 1 failed, 2 passe
```

--- a/rbc/irtools.py
+++ b/rbc/irtools.py
@@ -241,7 +241,8 @@ def compile_to_LLVM(functions_and_signatures, target: TargetInfo,
     flags = compiler.Flags()
     flags.set('no_compile')
     flags.set('no_cpython_wrapper')
-    flags.set('no_cfunc_wrapper')
+    if get_version('numba') >= (0, 49):
+        flags.set('no_cfunc_wrapper')
 
     function_names = []
     for func, signatures in functions_and_signatures:

